yosh/builtins/history.py

```diff
@@ -0,0 +1,19 @@
+import os
+from yosh.constants import *
+
+
+def history(args):
+    count = 0
+    if args:
+        count = int(args[0]) + 1
+    file = open(os.getenv("HOME") + "/.shistory")
```

Comment:
- Change file name to `.yosh_history`  (like `.bash_history`, `.zsh_history`).
- Use syntax `with open(os.getenv('HOME') + '/.yosh_history') as file:` to close file handler automatically.